tests/e2e/fill-incorrect-identity-input-values.test.ts

```diff
@@ -48,11 +52,7 @@ describe("Fill incorrect identity input values", () => {
 
   describe("AddIdentity", () => {
     test("It should show error messages if Identity inputs are filled incorrectly in add identity", async () => {
-      expect.assertions(9);
```

Comment:
why?

Reply:
Two reasons:
- For now, the function `authenticateToConnect` has some `expect` inside and we must add them in this count for the test to pass. I find that it make it useless because not reflecting the number of "real assertions" in the test.
- It's useful when something can pass or not, like with errors or calls or whatever that can "silently" fail, but here, we expect some things that will make the test fail if there are not there anyway. For example, we expect some input before trying to write inside. If the input is not there, the write action will fail and make the test fail in the same time.

Reply:
> It is an absolute hell to try our test suites with expect.assertions. And I'll find a better way to handle this but it's not for this PR

I'm curious on this point.

I would find it better to remove the expects from the `authenticateToConnect` since we don't need to test that though.

Reply:
> It is an absolute hell to try our test suites with expect.assertions

Not talking about code here but we have many `expect`, and as I "play" with the tests and move things, I had to keep the count in mind and it was pretty annoying.

> I would find it better to remove the expects from the authenticateToConnect

I tried to not do that in this PR to keep changes  at a minimum because if I do, I have to change every `expect.assertions` for e2e tests.

As you prefer!

Reply:
> Not talking about code here but we have many expect, and as I "play" with the tests and move things, I had to keep the count in mind and it was pretty annoying.

You definition of "absolute hell" is quite a gentle one 😅 . Just comment the line while you're doing your tests and add it back at the end :)
But it's really a good practice to have it,I think it can save our beacon even in e2e tests.

> I tried to not do that in this PR to keep changes at a minimum because if I do, I have to change every expect.assertions for e2e tests. But can do, it's as you prefer!

Well you have my blessing 😇